### PR TITLE
Add new framework: Fiber

### PR DESCRIPTION
**Pull request by Fenny**
Hello,

I would like to add [Fiber](https://github.com/fenny/fiber) to the go-web-framework-benchmark.
Passed all tests locally, let me know if I'm missing something.

Thank you!

PS: *@smallnest*, what an awesome project!

### CODE REVIEW

File: libs.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
 web_frameworks=("default" "atreugo" "beego" "bone" "chi" "denco" "echov3" "fasthttp-raw" \
-"fasthttp-routing" "fasthttp/router" "fasthttprouter" "fastrouter" "fresh" "gear" "gin" "goji" "gojsonrest" "gongular" "gorestful" "gorilla" "gorouter" "gorouterfasthttp" "go-ozzo" "gowww" "gramework" "httprouter" \
+"fasthttp-routing" "fasthttp/router" "fasthttprouter" "fastrouter" "fresh" "fiber" "gear" "gin" "goji" "gojsonrest" "gongular" "gorestful" "gorilla" "gorouter" "gorouterfasthttp" "go-ozzo" "gowww" "gramework" "httprouter" \
```
Comment (smallnest) [inline libs.sh]: 这里框架的名称是按字母顺序进行排序的

File: server.go [hunk 2]
```diff
@@ -149,6 +150,8 @@ func main() {
 		startFastRouter()
 	case "fresh":
 		startFresh()
+	case "fiber":
```
Comment (smallnest) [inline server.go:151]: 框架按字母顺序排序

Comment (Fenny) [reply server.go:151]: Edited

File: libs.sh (hunk 1 above)
Comment (Fenny) [reply libs.sh]: Edited

**Comment by Fenny** [comment]
Updated the alphabetic order

**Pull request merged**